endpoint-deprecation/unique_req_header_values/vis.json

```diff
@@ -0,0 +1,64 @@
+{
+  "variables": [
+      {
+          "name": "start_time",
+          "type": "PX_STRING",
+          "description": "The relative start time of the window. Current time is assumed to be now",
+          "defaultValue": "-5m"
+      },
+      {
+          "name": "service",
+          "type": "PX_SERVICE",
+          "description": "The name of the service to get stats for.",
+          "defaultValue": "default"
+      },
+      {
+          "name": "endpoint",
+          "type": "PX_STRING",
+          "description": "The endpoint to look at stats for. Format: /path/to/endpoint/*/more/path ",
+          "defaultValue": "/"
+      },
+      {
+          "name": "max_num_records",
+          "type": "PX_INT64",
+          "description": "Max number of records to show.",
+          "defaultValue": "1000"
+      }
```

Comment:
i might just drop this limit altogether since it might give the impression it's a complete list - tho we cap it to 10,000 no matter what anyway unless you specifically override it.